lib/Profile.dart

```diff
@@ -128,7 +148,8 @@ class _ProfileState extends State<Profile> {
                                           context,
                                           MaterialPageRoute(
                                               builder: (context) =>
-                                                  EditProfile(riderProvider.info)));
+                                                  EditProfileName(
+                                                      riderProvider.info)));
```

Comment:
In the future, we might not need to go to a completely new page to edit a rider's name. For now, it can stay because  the functionality is there.